src/main/webapp/mapScript.js

```diff
@@ -12,18 +12,36 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+
+
 function initMap(mapLocation) {
   let myLatLng = {lat: mapLocation.lat, lng: mapLocation.lng};
   map = new google.maps.Map(document.getElementById('map'), {
-    zoom: 13,
     center: myLatLng,
     mapTypeControl: false
   });
 }
 
-function createMarker(listing) {
+function createMarker(listing, cardNumber) {
   let marker = new google.maps.Marker({
     position: {lat: listing.mapLocation.lat, lng: listing.mapLocation.lng},
   });
+  bounds.extend(marker.getPosition());
+
+  marker.addListener('click', function() {
+    map.setCenter(marker.getPosition());
+    map.setZoom(18);
```

Comment:
Confirming this is meant to be different from the original zoom value. 18 != 13

Reply:
Yes -- @AbakirH found a function where you can fit the bounds of the map to the markers so the initial map should be set to the best fit zoom and then it zooms more when a marker is clicked.